### PR TITLE
NTNet Nuking, Improved PDA Messenger!

### PR DESCRIPTION
**Pull request by RimiNosha**
## About The Pull Request

So, this started as just porting over the improved PDA messenger, but then I was reminded by merge conflicts that NTNet is a jank piece of shit, so I decided to nuke that too!

Ports:
- https://github.com/tgstation/tgstation/pull/75820
- https://github.com/tgstation/tgstation/pull/74085
- https://github.com/tgstation/tgstation/pull/74142
- https://github.com/tgstation/tgstation/pull/71735
- https://github.com/tgstation/tgstation/pull/76215
- And probably a few bugfix PRs I forgot about.

## How Does This Help ***Gameplay***?

NTNet causes funny little bugs like door remotes being broken if spawned on a z without NTNet. Also speeds up init by almost a second, which is neat.

New NTNet messenger makes the messenger far more usable, and photos are now actually feasible to use.

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary> 

image
image
image
</details>

## Changelog

:cl: distributivgesetz, JohnFulpWillard (original), RimiNosha (port)
del: NTNet is gone.
qol: The PDA messenger has been overhauled. Enjoy!
/:cl:

### CODE REVIEW

File: code/controllers/subsystem/processing/networks.dm [hunk 1]
```diff
@@ -1,483 +0,0 @@
...
-	add_log("Configuration Updated. Now keeping [setting_maxlogcount] logs in system memory.")
-
-
-
-/**
- * Gives an area a root and a network_area_id
- *
- * When a device is added to the network on map load, it needs to know where it is.
- * So that it is added to that ruins/base's network instead of the general station network
- * This way people on the station cannot just hack Charlie's doors and visa versa.  All area's
```
[the remark this answers is not in the document]
Comment (RimiNosha) [reply code/controllers/subsystem/processing/networks.dm:241]: Added a helper.

**Comment by RimiNosha** [comment]
Should be like 99% done, just need to verify things that used to use NTNet still actually work

**Comment by QuacksQ** [comment]
image

Are these blue text boxes switched to orange?

**Comment by RimiNosha** [comment]
Good point, that's awful readability.

**Comment by RimiNosha** [comment]
Worked perfectly during the playtest, barring pathfinder shuttles not working properly with remotes, which is now fixed.

**Pull request merged**